### PR TITLE
Fix issue with missing scene covers in SARJ scraper

### PR DESCRIPTION
**Pull request by DampToast**
The API returned links to covers that didn't exist.  This checks if they exist before passing the image link onwards.

### CODE REVIEW

**Comment by bnkai** [comment]
This looks good. Does it make sense to actually download, b64 encode and pass through the actual image since we are already doing the request?

File: scrapers/SARJ-LLC.py [hunk 1]
```diff
@@ -196,7 +196,23 @@ def scrape_movie(base_url, date, name):
...
+            res['Image'] = f"https://www.{studio[1]}{image_part}"
+            try:
+                response = requests.get(res['Image'], headers={
+                    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0'
+                }, timeout=(3, 6))
+            except requests.exceptions.RequestException as e:
+                log.error(f"Error fetching URL {res['Image']}: {e.strerror}")
+
+            if response.status_code < 400:
+                break
```
Comment (bnkai) [inline scrapers/SARJ-LLC.py:212]:
```python
                if response.status_code < 400:
                    res['Image'] = f"data:image/jpeg;base64,{base64.b64encode(response.content).decode()}"
                    response.close()
                    break
```
Something like that seems to work (you will need to import `base64` and for some reason remove the `log.debug(f"Send output: {output}"` line at the end of the scraper)

Comment (DampToast) [reply scrapers/SARJ-LLC.py:212]: I figured it out then came back here and found it was already done.  I'll put amend the PR shortly

**Comment by DampToast** [comment]
Should be the better way now.  Also just did some auto-formatting

**Pull request merged**